autoscale/src/main/java/com/sequenceiq/periscope/service/AltusMachineUserService.java

```diff
@@ -47,7 +48,7 @@ public class AltusMachineUserService {
     private CloudbreakVersionService cloudbreakVersionService;
 
     @Inject
-    private ClusterService clusterService;
+    private ClusterRepository clusterRepository;
```

Comment:
why did you change this? as a rule of thumb we should have the repository used/injected in the respective service, and every other service should use it

Reply:
To straigthen out a circular dependence

Reply:
This change is reverted because of the allow-circular-dependency flag allows the service to start.